apps/rush-lib/src/logic/PackageChangeAnalyzer.ts

```diff
@@ -56,10 +58,32 @@ export class PackageChangeAnalyzer {
     let repoDeps: IPackageDeps;
     try {
       if (this._isGitSupported) {
+        const commandLineConfigFile: string = path.join(
+          this._rushConfiguration.commonRushConfigFolder,
+          RushConstants.commandLineFilename
+        );
+
+        const commandLineConfiguration: CommandLineConfiguration = CommandLineConfiguration.loadFromFileOrDefault(
+          commandLineConfigFile
+        );
+
+        // Generate possible package-deps filenames from commands in order to
+        // ensure they are not included in the repoDeps
+        const excludes: string[] = commandLineConfiguration.commands.reduce(
+          (filenames: string[], command: CommandJson) => {
+            if (command.commandKind === 'bulk') {
+              filenames.push(`package-deps.${command.name}.json`);
```

Comment:
Extract the `package-deps.${command.name}.json` into a function so that this won't be defined in multiple places.

Reply:
Like this?

```suggestion
  public getPackageDepsFilename(name: string): string {
    return `package-deps.${name}.json`;
  }
```

```suggestion
    filenames.push(this.getPackageDepsFilename(command.name));
```